src/config.rs

```diff
@@ -155,3 +158,50 @@ create_config! {
     cfg_test: bool, true, false, "build cfg(test) code";
     unstable_features: bool, false, false, "enable unstable features";
 }
+
+/// A rustfmt config (typically specified via rustfmt.toml)
+pub struct FmtConfig(rustfmt::config::Config);
```

Comment:
Could you document why you are using a newtype, rather than just the rustfmt config directly please